src/style.css

```
@@ -1,3 +1,104 @@
+/* stylelint-disable font-family-name-quotes */
+/* stylelint-disable-next-line comment-empty-line-before */
+
+:root {
+  --blue: #00acdc;
+  --header-height: 60px;
+}
+
+.roboto-bold {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+}
+
+.moyo_head {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  text-transform: uppercase;
+  align-items: center;
+  height: var(--header-height);
+}
+
+.navi_box {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0;
+  margin-right: 50px;
+  align-content: center;
+  height: var(--header-height);
+  margin-left: 0;
+  padding-left: 0;
+  box-sizing: border-box;
+}
+
+.navi_list {
   margin: 0;
+  padding: 0;
+  height: var(--header-height); /* 48px to make it equal to test 3-4 */
+  line-height: var(--header-height); /* 36px to make it equal to test 3-4 */
+}
+
+.navi_link {
+  display: inline-block;
+  height: var(--header-height); /* 48px to make it equal to test 3-4 */
+  line-height: var(--header-height); /* 36px to make it equal to test 3-4 */
+}
+
+.link {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  height: var(--header-height); /* 48px to make it equal to test 3-4 */
+  line-height: var(--header-height); /* 36px to make it equal to test 3-4 */
+}
+
+.link:hover {
+  color: var(--blue);
+}
+
+li:not(:first-child) {
+  margin-left: 8.5px;
+}
+
+li:not(:last-child) {
+  margin-right: 8.5px;
+}
+
+li.is-active {
+  color: var(--blue);
+  position: relative; /* Added for positioning context */
```

Remove comments everywhere
```suggestion
  position: relative;
```